Formula/hledger.rb

```diff
@@ -3,27 +3,41 @@
 class Hledger < Formula
   include Language::Haskell::Cabal
 
+  hledger_version = "1.11.1"
+  hackage_base = "https://hackage.haskell.org/package"
+
   desc "Command-line accounting tool"
   homepage "http://hledger.org"
-  url "https://hackage.haskell.org/package/hledger-1.11.1/hledger-1.11.1.tar.gz"
+  url "#{hackage_base}/hledger-#{hledger_version}/hledger-#{hledger_version}.tar.gz"
```

Comment:
This will make use of `brew bump-formula-pr` harder. Please don't use interpolation here.